blog/2023-year-in-review.md

```diff
@@ -0,0 +1,105 @@
+
+Here are some quick statistics from our Brown community made in 2023. We had 1557 active Oscar users, which is a 24.66% increase in activity from the year prior. Over 3.6 million jobs were run in Oscar totalling over 57 million CPU core hours! Since 2018, the number of active oscar users has increased and well as demand for resources. Over **470** questions were asked and helped in our weekly office hours.
+
+<div style="text-align: center;">
+    <img 
+        src="/content/images/blog/2023-year-in-review/active-oscar-users-in-2023.png" 
+        alt="Active Oscar Users in 2023"
+        style="border: 5px solid #555; width: 400px"
+    />
+    <img 
+        src="/content/images/blog/2023-year-in-review/oscar-jobs.png" 
+        alt="Oscar Jobs"
+        style="border: 5px solid #555; width: 400px"
+    />
+    <img 
+        src="/content/images/blog/2023-year-in-review/cpu-core-hours.png" 
+        alt="CPU Core Hours"
+        style="border: 5px solid #555; width: 400px"
+    />
+</div>
+
+
+<div style="text-align: center;">
+    <figure>
+        <img 
+            src="/content/images/blog/2023-year-in-review/active-oscar-users.png" 
+            alt="Active Oscar Users"
+            style="border: 5px solid #555; width: 400px"
+        />
+        <figcaption>This bar chart displays an increase of active Oscar users since 2018.</figcaption>
+    </figure>
+</div>
+
+In June, Oscar transitioned from General Parallel File System (GPFS) to an all-flash parallel filesystem (**VAST data**) as its primary storage pool. And furthermore, it was done without any major issues. This is arguably the biggest change Oscar has gone through in the last 10 years!
+
+Alongside supporting Oscar for high-performance computing needs, CCVers provided academic support to the Brown community. We hosted 17 Data Science, Computing, and Visualization Workshops (DSCoVs) with the Data Science Initiative (DSI). Topics included Gene Annotation Resources in R, ChatGPT's API and Prompt Engineering, Building VR Application in Unity, and more! Jupyterhub, cloud-hosted Jupyter Notebooks for multiple users, supported *8* undergraduate and graduate courses. In June, we hosted a bootcamp, a series of online/hybrid tutorials, wherein *over 175* members of the Brown community signed up to learn about research computing resources and get hands-on practice using Oscar.
+
+
+In 2023, we supported 60 projects and collaborations throughout Brown. The following are some highlights from those projects.
+
+- We launched our first mobile applications: [**SOMA**](https://somatheapp.com/), an app focused on pain management, and [**MAPPS**](https://www.mappsproject.com/), an app designed to explore social interactions and their influence on disease transmission patterns.
+- [**Honeycomb**](https://brown-ccv.github.io/honeycomb-docs/), a template for reproducible psychological tasks for clinic, laboratory, and home use has undergone a significant upgrade and has released version 3.2.6.
```

Comment:
@eldu here's my best sentence for Honeycomb! Not sure how much better it is haha
```suggestion
- [**Honeycomb**](https://brown-ccv.github.io/honeycomb-docs/) released versions 3.1 and 3.2 which better align the repository with [jsPsych](https://www.jspsych.org/7.3/) best practices.
```